distribution/docs/src/main/jdocs/content/_configuring/environment-hardening-contents.adoc

```diff
@@ -45,11 +45,18 @@ On system: ensure that not everyone can change ACLs on your object.
 
 |SSH
 |tampering, information disclosure, and denial of service
-a|Create accounts for authentication (modify the default user's password) or remove ssh feature. By definition the connection will be secure when authenticated. Optionally, disable the ssh port (`default: 8101`) and set the `karaf.startRemoteShell` system property to false.
+a|By default, SSH access is disabled in ${branding}. To turn it on, edit the `etc/org.apache.karaf.shell.cfg`
```

Comment:
Is a restart required after editing this config file?

Reply:
No restart is required; I'll indicate that in the docs.